src/collective/cover/tests/test_carousel_tile.py

```diff
@@ -37,7 +39,14 @@ def test_default_configuration(self):
         self.assertTrue(self.tile.is_editable)
 
     def test_accepted_content_types(self):
-        self.assertEqual(self.tile.accepted_ct(), ALL_CONTENT_TYPES)
+        # Add folder to default accepted_ct
+        registry = getUtility(IRegistry)
+        settings = registry.forInterface(ICoverSettings)
+        settings.searchable_content_types += ['Folder']
```

Comment:
Actually, if you inherit from base, you don't need to override `accepted_ct` to add a new valid accepted type, you can do like @rodfersou did:

https://github.com/collective/collective.cover/blob/2956dfc5a5c3c87af48efd679e0ce3391dca7ed9/src/collective/cover/tiles/base.py#L160

Is a matter of "taste", I suppose.